src/main/java/command/ManageCommand.java

```diff
@@ -20,17 +20,18 @@ public class ManageCommand extends Command {
     /**
      * The index of the task to be managed.
      */
-    private int index;
+    private int[] indices;
 
     /**
      * Constructs a ManageCommand with the specified command type and index.
      *
      * @param type  The type of manage command (MARK, UNMARK, DELETE).
-     * @param index The index of the task to be managed.
+     * @param indices The index of the task to be managed.
      */
-    public ManageCommand(Command.Types type, int index) {
+    @SafeVarargs
+    public ManageCommand(Command.Types type, int... indices) {
         this.type = type;
-        this.index = index;
+        this.indices = indices;
```

Comment:
Great distinction to separate the use of the word index and the comment! The small things in readability add up 👍